build/root/usr/local/bin/run

```diff
@@ -5,9 +5,7 @@
 #
 # SOURCE and HOME DIRECTORY: /opt/ci-artifacts/src
 
-set -o nounset
 set -o errexit
-set -o pipefail
```

Comment:
why did you remove these flags? 

> Use `set -o pipefail` in scripts to catch mysqldump fails in e.g. mysqldump |gzip. The exit status of the last command that threw a non-zero exit code is returned.

> Use `set -o nounset` (a.k.a. `set -u`) to exit when your script tries to use undeclared variables.

except if there is a very good reason behind it, better keeping them I think

Reply:
https://github.com/openshift-psap/ci-artifacts/pull/17/commits/1dd4636644489e884e8785d21aea01cf3d9f0470